CHANGELOG.md

```diff
@@ -2,6 +2,29 @@
 
 ## Unreleased
 
+## v0.67.0
+
+This Splunk OpenTelemetry Collector release includes changes from the [opentelemetry-collector v0.67.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.67.0) and the [opentelemetry-collector-contrib v0.67.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.67.0) releases where appropriate.
+
+### 🛑 Breaking changes 🛑
+- Restrict Collector service command line arguments [#2343](https://github.com/signalfx/splunk-otel-collector/pull/2343) 
```

Comment:
Can someone please show me any use case that is "broken" to call it  🛑 Breaking changes 🛑 ? :)

Reply:
We don't need to stop the release. I can prepare a PR to remove this unless anyone can show me what is broken. How does it sound?